app/templates/components/item-summary.hbs

```diff
@@ -0,0 +1,67 @@
+<section class="move-item-to-box">
```

Comment:
I believe this component already exist in the Actions UI built by @swatijadhav. Could you make sure there's no duplicate component ? You can probably use this component over there.

Also, a couple points

* `class="move-item-to-box"` makes little sense
* the name is very misleading, it's a `package` and not an `item`, and `"summary"` could mean anything. So I'd recomment `package-summary-block` for both the component name and the CSS class name

Reply:
PS: I know Stock uses the `item` wording instead of `package`. But it's wrong, and we need to start reversing that

Reply:
Swati's UI is different than this. It is similar to box pallet UI. There was just HTML for that, I used that HTML (with modifications)  in this component.  
Surely, i'll change the CSS names.